scheduler/kubernetes/scheduler-convergence.md

```diff
@@ -53,3 +56,15 @@ spec:
         persistentVolumeClaim:
           claimName: mysql-data
 ```
+
+### Initializer (Experimental feature in STORK v1.1)
+If you are not able to update the schedulerName for you applications to use
+stork, you can enable the app-initializer feature. This uses the Kubernetes
+[AdmissionController Initializer](https://kubernetes.io/docs/admin/extensible-admission-controllers/#initializers)
+feature to automatically update the scheduler to stork if your application
+(deployment or statefulset) is using volumes backed by Portworx.
+
+To enable the Initializer you need to:
+* [Enable the Intializer feature in your Kubernetes cluster](https://kubernetes.io/docs/admin/extensible-admission-controllers/#enable-initializers-alpha-feature) since it is an alpha feature in Kubernetes.
+* Add "--app-initializer=true" option in the stork deployment
+* Add the [stork-initializer spec](https://raw.githubusercontent.com/libopenstorage/stork/master/specs/stork-initializer.yaml) to you Kubernetes cluster using `kubectl create -f stork-initializer.yaml`
```

Comment:
change `kubectl create` to `kubectl apply` to avoid warnings on subsequent applies.

Reply:
Done